src/contracts/action-hubs/CappedTokenTransfersHub.sol

```diff
@@ -109,11 +109,14 @@ contract CappedTokenTransfersHub is ActionHub, ICappedTokenTransfersHub, SafeMan
 
```

Comment:
a note for the UI about this function is that it should consider soon-to-be-executed transfers if planning on queueing another one given the capLeft can be "unsynchronized", as an example if a capped transfer has not updated the state and an epoch has passed, it will assume it has the full cap, however, when the other transfer updates the state, that cap will be lower, which can lead to having to wait for the next epoch in the cases where the second transfer added to the first one surpasses the epoch cap